_update-tox-files

```diff
@@ -22,6 +22,9 @@ for charm in $charms; do
                 cp -fvp global/$charm_type/src/tox.ini charms/$charm/src/tox.ini
                 cp -fvp global/$charm_type/rename.sh charms/$charm/rename.sh
                 ;;
+            ops-unknown)
```

Comment:
I think we should fix whatever is generating the category name *ops-unknown* (it's probably the script `what-is`). I think we need ops-k8s and ops-machine or something similar